record.go

```diff
@@ -0,0 +1,149 @@
+package sarama
+
+import "encoding/binary"
+
+const (
+	controlMask = 0x20
+)
+
+type Header struct {
```

Comment:
should we call this `RecordHeader` to be a bit clearer?

Reply:
Ok